### ISSUE TITLE
JavaScript-Error when loading LabelFormFields (Ajax, Dialog)

### ISSUE THREAD

**Issue by mutec** [WoltLab/WCF#3078]
`$labelFields = LabelFormField::createFields('de.mysterycode.foo', $labelGroups);`

```
WCF.Combined.min.js?v=1567631518:18 Uncaught TypeError: Cannot read property 'sortable' of undefined
    at VM883 WCF.Combined.min.js:18
    at VM883 WCF.Combined.min.js:18
    at VM883 WCF.Combined.min.js:18
(anonymous) @ WCF.Combined.min.js?v=1567631518:18
(anonymous) @ WCF.Combined.min.js?v=1567631518:18
(anonymous) @ WCF.Combined.min.js?v=1567631518:18
```
The corresponding lines:
```
// 3rdParty/jquery-ui/nestedSortable.js
(function(window, undefined) {
    !function(t) {
        t.widget("mjs.nestedSortable", t.extend({}, t.ui.sortable.prototype, {
```

In debug-mode everything is working fine. Seems to be an issue when using the minified JS.

**Comment by Gravatronics** [comment]
Please check if 5c09fb6319632630b2f405f5875677714fd66ba2 fixes this issue and #3079.

**Comment by mutec** [comment]
It's fixed. Thanks.
